build.sbt

```diff
@@ -123,7 +123,7 @@ lazy val kg = project
       serviceKafka,
       serviceKamon,
       serviceHttp,
-      wesoSchema,
+      topQuadrantShacl,
```

Comment:
should also remove `resolvers  += Resolver.bintrayRepo("bogdanromanx", "maven")` as that's no longer required.